modules/ref/pages/client-settings.adoc

```diff
@@ -225,6 +225,12 @@ UserManager, CollectionManager, QueryIndexManager, etc.) if not overridden by a
 timeout. The default is quite high because some operations (such as flushing a bucket,
 for example) might take a long time.
 
+Name: *ServerWaitBackoff*::
```

Comment:
This should like server_wait_backoff to match what the connection string property should be. We should also make it clear that it's connection string only, see unordered_execution_enabled

Reply:
I thought this but none of the other settings here that are configurable through the connection string are like this, e.g KVTimeout vs kv_timeout. But is this because this is how they appear in the cluster config whereas server_wait_backoff doesn't appear in config?